src/applications/personalization/profile-2/components/connected-apps/actions/index.js

```diff
@@ -0,0 +1,67 @@
+import { apiRequest } from 'platform/utilities/api';
+
+export * from 'platform/user/profile/actions';
+
+import environment from 'platform/utilities/environment';
+
+export const LOADING_CONNECTED_APPS = 'connected-apps/LOADING_CONNECTED_APPS';
+export const FINISHED_CONNECTED_APPS = 'connected-apps/FINISHED_CONNECTED_APPS';
+export const ERROR_CONNECTED_APPS = 'connected-apps/ERROR_CONNECTED_APPS';
+export const DELETING_CONNECTED_APP = 'connected-apps/DELETING_CONNECTED_APP';
+export const ERROR_DELETING_CONNECTED_APP =
+  'connected-apps/ERROR_DELETING_CONNECTED_APP';
+export const FINISHED_DELETING_CONNECTED_APP =
+  'connected-apps/FINISHED_DELETING_CONNECTED_APP';
+export const DELETED_APP_ALERT_DISMISSED =
+  'connected-apps/DELETED_APP_ALERT_DISMISSED';
+
+const grantsUrl = '/profile/connected_applications';
+
+import { mockConnectedApps } from 'applications/personalization/profile360/util/connected-apps.js';
+
+export function loadConnectedApps() {
+  return async dispatch => {
+    dispatch({ type: LOADING_CONNECTED_APPS });
+
+    // Locally we cannot call the endpoint
+    if (environment.isLocalhost()) {
+      await new Promise((resolve) => setTimeout(resolve, 2000));
+      dispatch({
+        type: FINISHED_CONNECTED_APPS,
+        data: mockConnectedApps,
+      });
+      return mockConnectedApps;
```

Comment:
Does this action creator need to return anything? I don't think having it return something has an impact one way or another.

Reply:
You are correct! Removed.